tests/foreman/cli/test_satellitesync.py

```diff
@@ -982,26 +977,37 @@ def test_positive_export_import_redhat_cv(
         assert len(cv['versions']) == 1
         cvv = cv['versions'][0]
         # Verify export directory is empty
-        assert validate_filepath(target_sat, function_org) == ''
+        assert validate_filepath(target_sat, function_entitlement_manifest_org) == ''
         # Export cv
         export = ContentExport.completeVersion(
-            {'id': cvv['id'], 'organization-id': function_org.id}
+            {'id': cvv['id'], 'organization-id': function_entitlement_manifest_org.id},
+            timeout=7200000,
+        )
+        import_path = move_pulp_archive(
+            target_sat, function_entitlement_manifest_org, export['message']
         )
-        import_path = move_pulp_archive(target_sat, function_org, export['message'])
         exported_packages = Package.list({'content-view-version-id': cvv['id']})
         assert len(exported_packages)
 
         # importing portion
-        importing_org = make_org()
+        importing_org = target_sat.api.Organization().create()
         # check that files are present in import_path
         result = target_sat.execute(f'ls {import_path}')
         assert result.stdout != ''
-        target_sat.upload_manifest(importing_org['id'], interface='CLI', timeout=7200000)
+        target_sat.upload_manifest(
+            importing_org.id,
+            duplicate_entitlement_manifest,
+            interface='CLI',
+            timeout=7200000,
+        )
+        importing_org.sca_disable()
```

Comment:
why disable SCA here?

Reply:
Same reason as above, this test was written using entitlement mode and I have defaulted to not changing to SCA in converting tests to manifester.